RecoTracker/MkFitCore/BuildFile.xml

```diff
@@ -2,6 +2,10 @@
 <use name="json"/>
 <use name="tbb"/>
 <flags CXXFLAGS="-fopenmp-simd"/>
+<architecure name="el8_amd64_.*"/>
```

Comment:
architecture is missing a ‘t’, and the tag should not have a trailing ‘/‘ since it is a container

Reply:
fixed